spec.html

```diff
@@ -43597,9 +43597,10 @@ <h1>Promise Resolve Functions</h1>
             1. If IsCallable(_thenAction_) is *false*, then
               1. Perform FulfillPromise(_promise_, _resolution_).
               1. Return *undefined*.
-            1. Let _thenJobCallback_ be HostMakeJobCallback(_thenAction_).
-            1. Let _job_ be NewPromiseResolveThenableJob(_promise_, _resolution_, _thenJobCallback_).
-            1. Perform HostEnqueuePromiseJob(_job_.[[Job]], _job_.[[Realm]]).
+            1. Let _resolvingFunctions_ be CreateResolvingFunctions(_promise_).
+            1. Let _thenCallResult_ be Completion(Call(_then_, _resolution_, &laquo; _resolvingFunctions_.[[Resolve]], _resolvingFunctions_.[[Reject]] &raquo;)).
+            1. If _thenCallResult_ is an abrupt completion, then
+              1. ! Call(_resolvingFunctions_.[[Reject]], *undefined*, &laquo; _thenCallResult_.[[Value]] &raquo;).
```

Comment:
```suggestion
              1. Perform ! Call(_resolvingFunctions_.[[Reject]], *undefined*, &laquo; _thenCallResult_.[[Value]] &raquo;).
```

Reply:
I keep getting "This diff has recently been updated. Refresh and try again.", except refreshing doesn't help. I applied locally and pushed.